pkgs/development/misc/resholve/default.nix

```diff
@@ -1,9 +1,18 @@
 { callPackage
-, doCheck ? true
+, ...
 }:
 
+let
+  source = callPackage ./source.nix { };
```

Comment:
[nit feel free to ignore]
The source path is only used here, how about putting the content on the file directly here ?

Reply:
I'm still ~fumbling for/towards a workflow that is easy to manage for local/out-of-nixpkgs work and in-tree (i.e., I would like to eventually fully automate updating the nixpkgs copy, but for now I manually diff and port changes). 

`source.nix` differs in the resholve repo to aid local development, so I'm just ~abstracting that perpetual source of difference  out of default.nix to make it easier to compare the nixpkgs/resholve copies.